chemclipse/plugins/org.eclipse.chemclipse.ux.extension.xxd.ui/src/org/eclipse/chemclipse/ux/extension/xxd/ui/swt/editors/ExtendedPCRPlateUI.java

```diff
@@ -75,7 +72,6 @@ private void createToolbarMain(Composite parent) {
 		comboSubsets = createComboSubsets(composite);
```

Comment:
When changing an element, the size needs to be adjusted if elements are added/removed:
`composite.setLayout(new GridLayout(4, false)); `